pandas/tests/io/test_clipboard.py

```diff
@@ -225,3 +225,16 @@ def test_invalid_encoding(self, df):
     @pytest.mark.parametrize('enc', ['UTF-8', 'utf-8', 'utf8'])
     def test_round_trip_valid_encodings(self, enc, df):
         self.check_round_trip_frame(df, encoding=enc)
+
+
+@pytest.mark.single
+@pytest.mark.clipboard
+@pytest.mark.skipif(not _DEPS_INSTALLED,
+                    reason="clipboard primitives not installed")
+class TestRawClipboard(object):
+
+    @pytest.mark.parametrize('data', [u'\U0001f44d...', u'Ωœ∑´...', 'abcd...'])
+    def test_raw_roundtrip(self, data):
+        import pandas.io.clipboard
+        pandas.io.clipboard.clipboard_set(data)
+        assert data == pandas.io.clipboard.clipboard_get()
```

Comment:
* Can you just import `clipboard_set` directly (and move it to the top)
* Let's just use test functions (instead of a class).  We are trying to be more `pytest`-idiomatic.
* Can you add your original issue example as a test to this file?